File: codespace/week02/05_scores_Arrays/scores.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Scores
    int scores[3];
    scores[0] = 72;
    scores[1] = 73;
    scores[2] = 33;

    // Print average
    printf("Average: %f\n", (scores[0] + scores[1] + scores[2]) / 3.0);

    int scores2[] = {72, 73, 33};
    for(int i = 0; i < 3; i++)
    {
        printf("score: %i \n", scores[i]);
    }
}

File: week06/070_lists_scores/scores.py
# # 1.)

# scores = [72, 73, 33]   #square brackets for lists in Python (not curly)

# average = sum(scores) / len(scores)
# print(f"Average: {average}")
# -----------------------------------
# 2.)
from cs50 import get_int

# scores = list()
scores = []
for i in range(3):
    score = get_int("Score: ")
    # scores.append(score)
    scores += [score]

average = sum(scores) / len(scores)
print(f"Average: {average}")
